Tests/AppTests/Mocks/BuildIndex+mock.swift

```diff
@@ -34,90 +34,96 @@ extension BuildIndex.Model {
                     name: "1.2.3",
                     kind: .release,
                     builds: [
-                        .init(id: Self.id, swiftVersion: .init(5, 2, 4), platform: .macosXcodebuild,    status: .pending),
-                        .init(id: Self.id, swiftVersion: .init(5, 2, 4), platform: .macosSpm,           status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 2, 4), platform: .macosXcodebuildArm, status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 2, 4), platform: .macosSpmArm,        status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 2, 4), platform: .ios,                status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 2, 4), platform: .tvos,               status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 1, 5), platform: .macosXcodebuild,    status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 1, 5), platform: .macosSpm,           status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 1, 5), platform: .macosXcodebuildArm, status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 1, 5), platform: .macosSpmArm,        status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 1, 5), platform: .ios,                status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 1, 5), platform: .tvos,               status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 0, 3), platform: .macosXcodebuild,    status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 0, 3), platform: .macosSpm,           status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 0, 3), platform: .macosXcodebuildArm, status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 0, 3), platform: .macosSpmArm,        status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 0, 3), platform: .ios,                status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(5, 0, 3), platform: .tvos,               status: .ok),
-                        .init(id: Self.id, swiftVersion: .init(4, 2, 3), platform: .macosXcodebuild,    status: .failed),
-                        .init(id: Self.id, swiftVersion: .init(4, 2, 3), platform: .macosSpm,           status: .failed),
-                        .init(id: Self.id, swiftVersion: .init(4, 2, 3), platform: .macosXcodebuildArm, status: .failed),
-                        .init(id: Self.id, swiftVersion: .init(4, 2, 3), platform: .macosSpmArm,        status: .failed),
-                        .init(id: Self.id, swiftVersion: .init(4, 2, 3), platform: .ios,                status: .failed),
-                        .init(id: Self.id, swiftVersion: .init(4, 2, 3), platform: .tvos,               status: .failed),
```

Comment:
This mock was actually quite outdated: the newest Swift we listed was 5.2 and that meant that the snapshots, which only cover the top of the page, showed all builds as pending, because the mock data contained no 5.3-5.5 builds.

I've updated the Swift versions used and shortened it to 3 Swift versions instead of 4.